src/modules/spaces/actions.js

```diff
@@ -57,6 +58,17 @@ export function fetchById(spaceId) {
         const user_id = value.user_id
         const has_user = !!(users.find(e => e.id === user_id))
         if (!has_user) { dispatch(userActions.fetchById(user_id)) }
+
+        // TODO(matthew): Right now, the space has an embedded user and organization record... why are we doing this
```

Comment:
Looking into it a bit, I'm pretty sure that came from before we had the embedded user, so is no longer needed